src/menus.jrl

```diff
@@ -321,7 +361,7 @@ p({
     "class":"foam.nanos.menu.ViewMenu",
     "view":{
       class: "foam.u2.borders.BaseUnAuthBorder",
-      children: [ {"class":"foam.u2.view.LoginView", "mode_":"SignIn"} ]
+      children: [ {"class":"foam.nanos.auth.login.LoginView", "mode_":0} ]
```

Comment:
It looks like you've completely replace the login view and associated signup/signin models.  Can we delete them LoginView and assocaited SignUp/SignIn models?

Reply:
in another PR.